src/eval.jl

```diff
@@ -54,7 +66,7 @@ handle("eval") do data
   lock(evallock)
   result = @dynamic let Media.input = Editor()
     withpath(path) do
-      @errs include_string(mod, text, path, line)
+      @errs include_string(mod, text, path, line) ends_with_semicolon(text)
```

Comment:
This feels unclean to me – `@errs` should only have one clear responsibility. Instead we could check whether the result should be suppressed just before rendering it, for example.